daemon/main.c

```diff
@@ -543,6 +543,8 @@ static void get_netdata_configured_variables() {
     // ------------------------------------------------------------------------
     // get default Database Engine page cache size in MiB
 
+    compute_multidb_diskspace();
```

Comment:
This function needs to be called after `default_rrdeng_disk_quota_mb` has been set below in line 561.

Reply:
True will use any user supplied value for the calculations